docs/src/main/paradox/routing-dsl/routes.md

```diff
@@ -184,3 +184,25 @@ Scala
 
 Java
 :   @@snip [RouteSealExample.java]($root$/src/test/java/docs/http/javadsl/RouteSealExample.java) { #route-seal-example }
+
+### Converting routes between Java and Scala DSLs
+
+In some cases when building reusable libraries that expose routes, it may be useful to be able to convert routes between
+their Java and Scala DSL representations. You can do so using the `asScala` method on a Java DSL route, or by using an
+`RouteAdapter` to wrap an Scala DSL route. 
```

Comment:
maybe reference the `asJava` method in there?

Reply:
ok :)